### PR TITLE
Reload the page when switching networks for sites that use web3

### PR DESCRIPTION
**Pull request by frankiebee**
Reloads the page when switching networks only for sites that use web3

### CODE REVIEW

**Comment by frankiebee** [comment]
fixes #873

**Comment by flyswatter** [comment]
Pulled and verified.  You made this ancient bug look easy!  Your network-controller work is huge!!!!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Current Master
 
+- Now when switching networks sites that use web3 will reload
```
Comment (flyswatter) [inline CHANGELOG.md]: Changelog note missing punctuation.

**Review by danfinlay — APPROVED** [review]
Awesome!

**Pull request merged**